source/clear-linux/tutorials/kubernetes/kubernetes-bp.rst

```diff
@@ -0,0 +1,104 @@
+.. _kubernetes-bp:
+
+Kubernetes Best Practices on |CL|
+#################################
+
+This tutorial shows you how to manage your Kubernetes cluster while using
```

Comment:
we may have other best practices in the future. perhaps move everything we have now under a "cluster update" sub section